.buildkite/longruns/pipeline.yml

```
@@ -141,4 +141,10 @@ steps:
         artifact_paths: "longrun_hs_rhoe_equil_dt250/*"
         agents:
           slurm_cpus_per_task: 8
+      
+      - label: ":computer: held suarez (ρe_tot) equilmoist noqthyper"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 400secs --disable_qt_hyper true --t_end 1000days --fps 30 --job_id longrun_hs_rhoe_equil_noqthyper --dt_save_to_sol 6hours --dt_save_to_disk 10days"
```

```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 400secs --disable_qt_hyperdiffusion true --t_end 1000days --fps 30 --job_id longrun_hs_rhoe_equil_noqthyper --dt_save_to_sol 6hours --dt_save_to_disk 10days"
```
(sorry, a typo in the previous comment)

---

oops! good catch! 🐛